src/sentry/api/endpoints/group_details.py

```diff
@@ -44,6 +44,11 @@ def _get_actions(self, request, group):
 
         action_list = []
         for plugin in plugins.for_project(project, version=1):
+            if plugin.slug in SHADOW_DEPRECATED_PLUGINS and not features.has(
```

Comment:
We host an instance of [Flagr](https://github.com/checkr/flagr) to check feature flags. Unfortunately, the service times out or otherwise crashes a non-trivial precent of the time. You should build your flags and checks to fallback to default behavior in that case.

Reply:
The fallback behavior should be to 404 users without perms, so all is good here